learntools/machine_learning/nbs/tut3.ipynb

```diff
@@ -0,0 +1,261 @@
+    "\n",
+    "To show you the techniques, we'll start by picking a few variables using our intuition. Later tutorials will show you statistical techniques to  automatically prioritize variables.\n",
+    "\n",
+    "To choose variables/columns, we'll need to see a list of all columns in the dataset. That is done with the **columns** property of the DataFrame (the bottom line of code below).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "melbourne_file_path = '../input/melbourne-housing-snapshot/melb_data.csv'\n",
+    "melbourne_data = pd.read_csv(melbourne_file_path) \n",
+    "print(melbourne_data.columns)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The Melbourne data has somemissing values (some houses for which some variables weren't recorded.)\n",
+    "# We'll learn to handle missing values in a later tutorial.  \n",
+    "# Your Iowa data doesn't have missing values in the predictors you use. \n",
+    "# So we will take the simplest option for now, and drop those houses from our data. \n",
+    "#Don't worry about this much for now, though the code is:\n",
+    "\n",
+    "# dropna drops missing values (think of na as \"not available\")\n",
+    "melbourne_data = melbourne_data.dropna(axis=0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are many ways to select a subset of your data. The [Pandas Course](https://www.kaggle.com/learn/pandas) covers these in more depth. But we can get our data set up for modeling with just two of these approaches.\n",
+    "\n",
+    "1. Dot notation, which we use to select the \"prediction target\"\n",
+    "2. Selecting with a column list, which we use to select the \"predictors\"\n",
+    "\n",
+    "## Selecting The Prediction Target \n",
+    "You can pull out a variable with **dot-notation**.  This single column is stored in a **Series**, which is broadly like a DataFrame with only a single column of data.  \n",
+    "\n",
+    "We'll use the dot notation to select the column we want to predict, which is called the **prediction target**. By convention, the prediction target is called **y**. So the code we need to save the house prices in the Melbourne data is"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "_cell_guid": "e5a6c21c-cda7-4df8-a9e9-15a2ae865026",
+    "_uuid": "599dc0855ff966c6e8d84b5bd447b315bf85e339",
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "y = melbourne_data.Price"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "a52b0b8d-ea0a-435b-bde1-1472f63977ba",
+    "_uuid": "a0fac0889241e77f2e2deefbd63bc9f057d0c826"
+   },
+   "source": [
+    "# Choosing Predictors\n",
```

Comment:
Maybe worth having a brief aside on terminology here? I'm imagining there might be readers who have some exposure to alternative terms like "features" or "independent variables", and it might take a while for it to click that what we're talking about here is the same thing.

Reply:
Also, whether or not you go into synonyms, I think you need some explanation of what we actually mean by "predictors". i.e. these are the inputs to our model, the variables it will consider when predicting the target variable (price). Our first simple decision tree model had one predictor, which was number of bedrooms. etc.